pkg/infrastructure/kogito_images.go

```diff
@@ -16,16 +16,26 @@ package infrastructure
 
 import (
 	"fmt"
-	"github.com/kiegroup/kogito-cloud-operator/version"
 	"strings"
+
+	"github.com/kiegroup/kogito-cloud-operator/version"
 )
 
 const (
 	versionSeparator = "."
 	// LatestTag the default name for latest image tag
 	LatestTag = "latest"
+	// DefaultImageRegistry the default services image repository
+	DefaultImageRegistry = "quay.io"
+	// DefaultImageNamespace the default services image namespace
+	DefaultImageNamespace = "kiegroup"
 )
 
+// GetDefaultImage returns the default full image of the image name.
+func GetDefaultImage(imageName string) string {
+	return DefaultImageRegistry + "/" + DefaultImageNamespace + "/" + imageName + ":" + GetRuntimeImageVersion()
```

Comment:
Please avoid this kind of concatenation on functions body, try this:

```suggestion
	return fmt.sprintf( "%s/%s/%s:%s", DefaultImageRegistry, DefaultImageNamespace, imageName, GetRuntimeImageVersion())
```
Test it before commiting since I wrote from the top of my head. xD

Reply:
Thanks for spotting. I missed to update this piece by using:

```
return framework.ConvertImageToImageTag(v1alpha1.Image{
		Domain:    DefaultImageRegistry,
		Namespace: DefaultImageNamespace,
		Name:      imageName,
		Tag:       GetRuntimeImageVersion(),
	})
```